lib/connect/ws.js

```diff
@@ -56,14 +57,20 @@ function createWebSocket (client, opts) {
 
   setDefaultOpts(opts)
   var url = buildUrl(opts, client)
-  return websocket(url, [websocketSubProtocol], opts.wsOptions)
+  // TODO (yodama): Add origin option to WebSocket connection to prevent Cross-Site WebSocket Hijacking
```

Comment:
>// TODO (yodama): Add origin option to WebSocket connection to prevent Cross-Site WebSocket Hijacking [](start = 2, length = 101)

Leaving a TODO in the code is counter intuitive.  Best case is you add it later which means you can add it now; worst case you never add it and this line loses meaning to everyone except you.  Either add the todo code or remove it the comment.